deployment/charts/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
+name: fuel-core
+description: Fuel Core Helm Chart
+type: application
+appVersion: "0.3.2"
```

Comment:
Could we add some sort of ci check that this version is kept in sync with `package.version` in `~/fuel-core/fuel-core/Cargo.toml`? 

Something similar to the `.github/workflows/scripts/verify_tag.sh` use of dasel, e.g.

```
HELM_APP_VERSION=$(cat deployment/charts/Chart.yaml | dasel -r yaml 'appVersion')
FUEL_CORE_VERSION=$(cat fuel-core/Cargo.toml | dasel -r tomt 'package.version')
if [ "$HELM_APP_VERSION" != "$FUEL_CORE_VERSION" ]; then
    err "fuel-core version $FUEL_CORE_VERSION, doesn't match helm app version $HELM_APP_VERSION"
    exit 1
else
  status "fuel-core version matches helm chart app version $HELM_APP_VERSION"
fi

```

Reply:
I have to think about it - it requires the appVersion to have an ENV associated with it and it requires the user or CI process to run a script to update the Chart.yaml which would entail some envsubst 

typically the helm Chart.yaml is just manually updated with each release by the code committer

Reply:
it should run on PR's and on master, that way if you make a branch to bump versions you'll be notified to bump this one too before it merges to master.

Reply:
ok i will add this to this PR- thanks